randovania/games/dread/json_data/Cataris.txt

```diff
@@ -4954,8 +4966,9 @@ Extra - asset_id: collision_camera_063
   * Layers: default
   > Tunnel to Diffusion Beam Room (top)
       Any of the following:
-          Grapple Beam or Use Spin Boost
+          Simple IBJ or Use Spin Boost
           Spider Magnet and Before Elun - Release X Parasites
+          Grapple Beam and Grapple Movement (Beginner)
```

Comment:
still need to list magnet for having grapple as a possibility since the magnet method only covers for before the X release

Reply:
Right I left it out because I thought it was redundant but it matters for trickless, good catch

Reply:
huh, another instance of something that is fixed on my local branch but somehow didn't get to the online version, i'll push it again and see what happens
![image](https://user-images.githubusercontent.com/97209074/187332579-813a9094-28e8-4be8-94be-f75741d03159.png)